proposals/simd/BinarySIMD.md

```diff
@@ -226,45 +226,45 @@ For example, `ImmLaneIdx16` is a byte with values in the range 0-15 (inclusive).
-| `v128.load16_lane`              |     `TBD`| m:memarg, i:ImmLaneIdx8  |
-| `v128.load32_lane`              |     `TBD`| m:memarg, i:ImmLaneIdx4  |
-| `v128.load64_lane`              |     `TBD`| m:memarg, i:ImmLaneIdx2  |
-| `v128.store8_lane`              |     `TBD`| m:memarg, i:ImmLaneIdx16 |
-| `v128.store16_lane`             |     `TBD`| m:memarg, i:ImmLaneIdx8  |
-| `v128.store32_lane`             |     `TBD`| m:memarg, i:ImmLaneIdx4  |
-| `v128.store64_lane`             |     `TBD`| m:memarg, i:ImmLaneIdx2  |
-| `i64x2.eq`                      |     `TBD`| -                        |
-| `i64x2.ne`                      |     `TBD`| -                        |
-| `i64x2.lt_s`                    |     `TBD`| -                        |
-| `i64x2.gt_s`                    |     `TBD`| -                        |
-| `i64x2.le_s`                    |     `TBD`| -                        |
-| `i64x2.ge_s`                    |     `TBD`| -                        |
-| `i64x2.all_true`                |     `TBD`| -                        |
-| `f64x2.convert_low_i32x4_s`     |     `TBD`| -                        |
-| `f64x2.convert_low_i32x4_u`     |     `TBD`| -                        |
-| `i32x4.trunc_sat_f64x2_s_zero`  |     `TBD`| -                        |
-| `i32x4.trunc_sat_f64x2_u_zero`  |     `TBD`| -                        |
-| `f32x4.demote_f64x2_zero`       |     `TBD`| -                        |
-| `f64x2.promote_low_f32x4`       |     `TBD`| -                        |
-| `i8x16.popcnt`                  |     `TBD`| -                        |
-| `i16x8.extadd_pairwise_i8x16_s` |     `TBD`| -                        |
-| `i16x8.extadd_pairwise_i8x16_u` |     `TBD`| -                        |
-| `i32x4.extadd_pairwise_i16x8_s` |     `TBD`| -                        |
-| `i32x4.extadd_pairwise_i16x8_u` |     `TBD`| -                        |
+| `v128.load32_zero`              |    `0x5c`| -                        |
+| `v128.load64_zero`              |    `0x5d`| -                        |
+| `i16x8.extmul_low_i8x16_s`      |    `0x9c`| -                        |
+| `i16x8.extmul_high_i8x16_s`     |    `0x9d`| -                        |
+| `i16x8.extmul_low_i8x16_u`      |    `0x9e`| -                        |
+| `i16x8.extmul_high_i8x16_u`     |    `0x9f`| -                        |
+| `i32x4.extmul_low_i16x8_s`      |    `0xbc`| -                        |
+| `i32x4.extmul_high_i16x8_s`     |    `0xbd`| -                        |
+| `i32x4.extmul_low_i16x8_u`      |    `0xbe`| -                        |
+| `i32x4.extmul_high_i16x8_u`     |    `0xbf`| -                        |
+| `i64x2.extmul_low_i32x4_s`      |    `0xdc`| -                        |
+| `i64x2.extmul_high_i32x4_s`     |    `0xdd`| -                        |
+| `i64x2.extmul_low_i32x4_u`      |    `0xde`| -                        |
+| `i64x2.extmul_high_i32x4_u`     |    `0xdf`| -                        |
+| `i16x8.q15mulr_sat_s`           |    `0x9c`| -                        |
```

Comment:
I believe it is 0x82 (i16x8.extmul_low_i8x16_s uses 0x9c)

Reply:
Thanks, fixed.